ion-schema-tests-runner/src/generator.rs

```diff
@@ -321,18 +321,15 @@ fn generate_preamble(root_dir_path: &Path) -> TokenStream {
 
         /// Asserts that a value is or is not valid for a given ISL type.
         fn __assert_value_validity_for_type(value_ion: &str, schema_id: &str, type_id: &str, expect_valid: bool) -> Result<(), String> {
```

Comment:
Do you happen to know why we're using a `__` prefix rather than, say, making a new submodule to house these functions?

Reply:
I wanted to make sure that we didn't have any name conflicts with a generated test case, so I added a `__` prefix. I could have added a new submodule, but that probably also would have needed something to reduce the likelihood of a name conflict with a generated test function.